### PR TITLE
Appveyor support

### PR DESCRIPTION
**Pull request by joshmoore**
See: https://github.com/PyTables/PyTables/issues/424

This migrates the http://ci.openmicroscopy.org/job/PyTables build to Appveyor. The PR should be built automatically as with travis (and be green!) for merge.

### CODE REVIEW

**Comment by joshmoore** [comment]
Closing and re-opening to trigger appveyor.

**Comment by andreabedini** [comment]
awesome, let's see if it builds ok, perhaps move the badges on the main README ?

**Comment by joshmoore** [comment]
Happy to eventually, just no while everything's red :smile:

**Comment by andreabedini** [comment]
appveyor is currently failing because the build takes [too long](https://ci.appveyor.com/project/PyTablesBot/pytables/build/1.0.7/messages) (more than 40m!) and it seems all the time is spent [installing dependencies](https://ci.appveyor.com/project/PyTablesBot/pytables/build/1.0.7).

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+install:
+  # this installs the appropriate Miniconda (Py2/Py3, 32/64 bit),
+  # as well as pip, conda-build, and the binstar CLI
+  - powershell .\\ci\\appveyor\\install.ps1
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+
+build: false
+
+test_script:
+  - "%CMD_IN_ENV% conda install --yes hdf5 numexpr cython"
```
Comment (andreabedini) [inline appveyor.yml]: great work @joshmoore, maybe add [`--quiet`](http://conda.pydata.org/docs/commands/conda-install.html) so the the progress bar doesn't litter the log?

**Comment by FrancescAlted** [comment]
Any good reason for merging this into forthcoming PyTables 3.2?  My opinion is that this should not be high priority for 3.2.

**Comment by joshmoore** [comment]
No need to merge. This is an attempt to replace the OME Windows build. Neither is working at the moment, though.

**Comment by FrancescAlted** [comment]
Currently we have continuous-integration/appveyor failing and hence it is a bit confusing to determine if some PR is passing tests successfully.  I would say that we should complete the appveyor support as soon as possible, or just disable it temporarily.

**Comment by FrancescAlted** [comment]
Great.  Thanks.

**Comment by tomkooij** [comment]
With some (a lot of) fiddling, I've got Appveyor CI working. I restarted from the current `develop` and opened a new PR: #569

**Pull request closed without merging**